js/character-counter.js

```diff
@@ -6,6 +6,14 @@
 		this.characterCounter.call( el );
 	}
 
+	var length = function( e ){
+		if( /Safari|Chrome/.test(window.navigator.userAgent) ){
```

Comment:
It might be worthwhile brainstorming a few ideas on this to avoid UA sniffing here.